src/Tools/dotnet-user-jwts/src/Helpers/JwtAuthenticationSchemeSettings.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal sealed record JwtAuthenticationSchemeSettings(string SchemeName, List<string> Audiences, string ClaimsIssuer)
+{
+    private const string AuthenticationKey = "Authentication";
```

Comment:
We don't need to centralize this schema now, but the source of truth for these constants should really be in our DefaultAuthenticationConfigurationProvider thingy

Reply:
Agreed! Tracked here: https://github.com/dotnet/aspnetcore/issues/41888